client/app/domain/email-obfuscation/index.js

```diff
@@ -0,0 +1,12 @@
+import angular from 'angular';
+
+import component from './email-obfuscation.component';
+import controller from './email-obfuscation.controller';
+
+const moduleName = 'domainEmailObfuscation';
+
+angular.module(moduleName, [])
+  .component('domainEmailObfuscation', component)
+  .controller('domainEmailObfuscationCtrl', controller);
```

Comment:
Shouldn't be this instead?

```suggestion
  .controller('DomainEmailObfuscationCtrl', controller);
```